### PR TITLE
Bounty page continued

### PR DESCRIPTION
**Pull request by villanuevawill**
Lots of bug fixes across the site. Also finishing up the bounty page.

### CODE REVIEW

File: src/containers/CreateBounty/CreateBountyForm.js [hunk 1]
```diff
@@ -359,7 +360,11 @@ class CreateBountyFormComponent extends React.Component {
                         validators.required,
                         (balance, allValues) => {
                           const valueField = allValues.fulfillmentAmount;
-                          if (valueField && balance < valueField) {
+                          if (
```
Comment (c-o-l-o-r) [inline src/containers/CreateBounty/CreateBountyForm.js]: convert to HTML quote

File: src/public-modules/Bounty/sagas.js [hunk 2]
```diff
@@ -223,28 +226,26 @@ export function* createBounty(action) {
...
-          { from: userAddress }
-        ]
+      const test = yield call(
+        promisifyContractCall(tokenContractClient.approve, {
+          from: userAddress
+        }),
+        config[network].standardBountiesAddress,
+        parseInt(contractBalance)
+      );
+      yield call(delay, 2000);
```
Comment (c-o-l-o-r) [inline src/public-modules/Bounty/sagas.js]: https://github.com/MetaMask/metamask-extension/issues/3425
sad_mj

File: src/public-modules/Bounty/sagas.js [hunk 3]
```diff
@@ -493,6 +491,84 @@ export function* increasePayout(action) {
...
+      );
+      const network = yield select(networkSelector);
+      yield call(
+        promisifyContractCall(tokenContractClient.approve, {
+          from: userAddress
+        }),
+        config[network].standardBountiesAddress,
+        addedBalance
+      );
+      yield call(delay, 2000);
```
Comment (villanuevawill) [inline src/public-modules/Bounty/sagas.js]: Referencing issue:
https://github.com/MetaMask/metamask-extension/issues/3425

Batching didn't work previously, and tagging them one after another also didn't work.

Comment (villanuevawill) [reply src/public-modules/Bounty/sagas.js]: @danfinlay FYI - batching not properly working with the approve then call function.

Comment (villanuevawill) [reply src/public-modules/Bounty/sagas.js]: Putting it back to delay.

**Pull request merged**